ReactAndroid/src/main/java/com/facebook/react/modules/location/LocationModule.java

```diff
@@ -158,7 +157,7 @@ public void startObserving(ReadableMap options) {
           (LocationManager) getReactApplicationContext().getSystemService(Context.LOCATION_SERVICE);
       String provider = getValidProvider(locationManager, locationOptions.highAccuracy);
       if (provider == null) {
-        emitError("No location provider available.");
+        emitError(PositionError.PERMISSION_DENIED, "No location provider available.");
```

Comment:
nit: Is there a reason to use `PERMISSION_DENIED` here rather than `POSITION_UNAVAILABLE`?